third-culture-software/bhima
# Removed 'Add Tag' from modal

Added support for controlling whether the "Add Tag" link is shown when using this bhTagSelect modal.   

To test this, 
  - go to Stock > Stock Lots
  - click the [Search] button
  - verify that the "Add Tag" is not shown after the "Tags" label:
![image](https://user-images.githubusercontent.com/62145/95399799-da586780-08bd-11eb-8d30-4c17fd797127.png)

  - Now try editing one of the lots.  You should see the "Add Tag" link.
